### PR TITLE
Release 22.1.0

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [22.1.0](https://github.com/theforeman/puppet-katello/tree/22.1.0) (2023-02-02)
```
Comment (ekohl) [inline CHANGELOG.md:3]: This should be 22.1.1 (and on line 5 the same).

**Review by ekohl — COMMENTED** [review]
Minor note, otherwise :+1:

**Comment by ekohl** [comment]
I just noticed the commit message too :(

**Comment by ehelms** [comment]
> I just noticed the commit message too :(

Blech, sorry about that.

**Pull request merged**